docs/creating-nodes/appearance.md

```diff
@@ -297,7 +297,7 @@ A key principle is the editor is not a dashboard for controlling your flows. So
 in general, nodes should not have buttons on them. The Inject and Debug nodes are
 special cases as the buttons play a role in the development of flows.
 
-The `button` property in its definition is used to describe the behaviour of the
```

Comment:
Just going to say there's nothing wrong with `behaviour` - it is the proper spelling for those of us not in America

Reply:
Ok